src/appmixer/slack/bundle.json

```diff
@@ -18,6 +18,11 @@
         "2.4.1": [
             "Add new components: SendDirectMessage and ListUsers.",
             "Add new components: AddReaction, DeleteReaction, ListReactions and GetMessageReactions."
+        ],
+        "3.0.0": [
+            "(breaking change) SendChannelMessage, SendPrivateChannelMessage: in component.json moved channelId from properties to inPort.",
```

Comment:
@ZbynekPelunek are we sure that this is a breaking change? @vtalas to confirm

Reply:
we moved the input from properties to input message, therefore it's a breaking change